.ci/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:20.04
+ARG _ARCH=x86_64
+ARG _UID=6213
+ARG _GID=101
+ARG _LOGIN=swx-jenkins
+ARG _HOME=/var/home/$_LOGIN
+RUN mkdir -p $_HOME
+RUN groupadd -f -g "$_GID" "$_LOGIN"
+RUN useradd -u "$_UID" -g "$_GID" -s /bin/bash -m -d ${_HOME} "$_LOGIN"
+RUN chown $_LOGIN $_HOME
+USER root
+
+ENV TZ=Israel
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt clean && \
+    apt update && \
+    apt upgrade -y && \
+    apt install -y python3 python3-pip apt-transport-https sqlite3 sudo git
+
+RUN apt-get update && \
+    apt-get install -y \
```

Comment:
why do you need to run  each time? ones is enough.
apt-get update
apt upgrade -y